pages/common/lt.md

```diff
@@ -0,0 +1,25 @@
+# lt
+
+> Localtunnel exposes your localhost to the world for easy testing and sharing! No need to mess with DNS or deploy just to have others test out your changes.
+> Great for working with browser testing tools like browserling or external api callback services like twilio which require a public url for callbacks.
+> More information: <https://github.com/localtunnel/localtunnel>.
+
+- Start tunnel in a specific port:
+
+`lt -p 8000`
```

Comment:
We use long argument format if is available:
```suggestion
`lt --port 8000`
```

Reply:
Done 👍